### ISSUE TITLE
Postgres crashing, only goalert is accessing it.

### ISSUE THREAD

**Issue by wtc-jeff** [target/goalert#2828]
**Describe the Bug:**

The postgresql database will crash and stop, the only application using the database is goalert. If the server is left running without goalert, the database does not crash.

**Steps to Reproduce:**
Leave goalert running

**Expected Behavior:**
The database should not crash

**Observed Behavior:**
After leaving the server running for an undetermined amount of time, the postgresql database will error and stop.

**Screenshots/Stack Traces:**

The last 1000 lines of the postgres log with query logging enabled.

[postgres-log.txt](https://github.com/target/goalert/files/10860926/postgres-log.txt)

**Application Version:**
Output of `goalert version` and/or version information from view-source on the UI.

`
$ goalert version
Version:   v0.30.1
GitCommit: f5f5313c11d0ea1f14fa1c3f703a116b9fbbd093 (clean)
BuildDate: 2022-11-22T11:12:10-05:00
GoVersion: go1.19.3 (gc)
Platform:  linux/amd64
Migration: auth-link-requests (#226)
`

Postgresql version:
postgresql-13:
  Installed: 13.9-0+deb11u1

**Comment by wtc-jeff** [comment]
I had found that as a suggestion on the internet yesterday as a possible cause, the server was deployed originally with 4GB of RAM, I upgrade yesterday to have 16GB of ram and it still crashes.

**Comment by mastercactapus** [comment]
Is there anything you can share about the environment that may be relevant?
- how often are you seeing this?
- rough number of users/schedules/services/open alerts?
- how many GoAlert instances are running?
- is this a recent problem or a new environment?
- metrics to see how much memory Postgres was actually using when it crashed?
- Any relevant GoAlert errors/logs?
- hints from system logs (e.g., was it in fact OOM killer?)
- anything else running on the Postgres machine that could be using memory?

**Comment by mastercactapus** [comment]
Thanks for the details; I'll experiment and see if I can reproduce anything today.

For logs, GoAlert just logs to stdout; if you created a systemd service file named `goalert.service`, the logs would be available with `journalctl -u goalert`

What kind of CPU resources does the machine have? I'm going to start low just to see, but it would be good to know.

**Comment by wtc-jeff** [comment]
I have 2 virtual CPU's assigned to the VM, no reports of having excessive cpu utilization have been logged.  I did some searching for OOM errors and postgresql and ran across some suggestions on postgresql's website: 

https://www.postgresql.org/docs/current/kernel-resources.html#LINUX-MEMORY-OVERCOMMIT

I've implemented these suggestions and restarted.  I'll post an update over the weekend to see if this helps.
